index.html

```diff
@@ -0,0 +1,178 @@
+    <link rel="stylesheet" href="capstone.css">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+                
+</head>
+<body>
+    <section class="home-page">
+        <nav class="home-nav">
+            <ul>
+                <li><i class="fab fa-facebook-f"></i></li>
+                <li><i class="fab fa-twitter"></i></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Log Out</a></li>
+            </ul>
+        </nav>
+        <header class="header">
+            <nav>
+                <i class="fas fa-bars hamburger"></i>
+            </nav>
+            <div class="desk-nav">
+                <i class="fas fa-globe">MOOC</i>
+                <ul>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#">Home</a></li>
+                    <li><a href="#">Blog</a></li>
+                    <li><a href="">News</a></li>
+                    <li><a href="#">Sponsors</a></li>
+                    <li class="call"><a href="#">Call us</a></li>
+                </ul>
+            </div>
+        </header>
+        <div class="mobile-menu">
+            <div class="time">
+                <h3>09:41</h3>
+                <ul class="icon-left">
+                    <li><i class="fas fa-signal"></i></li>
+                    <li><i class="fas fa-wifi"></i></li>
+                    <li><i class="fas fa-battery-full"></i></li>
+                </ul>
+            </div>
+            <i class="fas fa-times cancel"></i>
+            <ul class="menu-list">
+                <li><div class="box"><a href="#work">Portfolio</a></div></li>
+                <li><div class="box"><a href="#about">About</a></div></li>
+                <li><div class="box"><a href="#contact">Contact</a></div></li>
+            </ul>
+        </div> 
+        <div class="head">
+            <div class="header-headings">
+                <h3 class="header-one">An Online Education for All!</h3>
+                <h1 class="header-two">MASSIVE OPEN ONLINE COURSE SUMMIT</h1>
+            </div>
+            <div class="header-para">
+                <p>A Global online conference that discusses all possible approaches to adopt for the Online Education Program.</p>
+            </div>
+            <div class="header-cont">
+                <h2 class="para-time">2022.04.15(Thurs) ~ 16(Fri)</h2>
+                <p class="para-contact">@globalonlineprogram.com</p>
+            </div>   
+        </div>
+        <section class="main">
+            <h3 class="program">Main Program</h3>
+            <div class="big-forum">
+                <div class="forum">
+                    <i class="fas fa-laptop"></i>
+                    <h3>Lecture</h3>
```

Comment:
- [ ] For the desktop size `MainProgram` section please make sure you stick to the design by adding the bottom button and title underline, colors, and fonts. Take a look at the below images to compare and fix the issues 👇 
- Original design 👇 
![Screenshot (187)](https://user-images.githubusercontent.com/67087664/153599204-89db9134-0057-4804-8fda-81880fd30ef8.png)

- Your design 👇 
![Screenshot (186)](https://user-images.githubusercontent.com/67087664/153599175-817aac96-e8e6-4ce6-b032-ae1f603d2ded.png)

Reply:
- [ ] Please make this change from the previous reviewer. currently, your design looks the same as it did before. make sure your design matches the Figma as closely as possible. 
- [ ] ensure on the page that you are using the correct fonts. those fonts are `COCOGOOSE` and `Lato`

Reply:
- [ ] Please make this change from the previous reviewer. currently, your design looks the same as it did before. make sure your design matches the Figma as closely as possible. 
- [ ] ensure on the page that you are using the correct fonts. those fonts are `COCOGOOSE` and `Lato`